languages/fsharp/exercises/concept/dates/.docs/hints.md

```diff
@@ -19,8 +20,14 @@
 - The tests are running as if running on a machine in the United States, which means that when converting a `DateTime` to a `string` will return dates and time in US format.
 - There is a [built-in method][to-string] to convert a `DateTime` instance to a `string`. If you are using `sprintf`, instead of using the `%s` string placeholder, there is [another placeholder][object-placeholder] that will automatically call the aforementioned method.
 
+### 5. Return the anniversary date
+
+- Use one of the various `DateTime` [constructors][constructors] to create a new `DateTime` instance.
+- You can use one of the current date time`s [properties][properties] to get the current year.
```

Comment:
This should be an apostrophe instead of a backtick
```suggestion
- You can use one of the current date time's [properties][properties] to get the current year.
```

Reply:
Good catch!